lib/build.py

```diff
@@ -360,7 +360,14 @@ def __init__(self, *args):
          self.dst = paths[-1]
       else:
          assert (ch.tree_child(self.tree, "copy_list") is not None)
-         self.unsupported_yet_fatal("list form", 784)
+         paths = [variables_sub(i, env.env_build)
```

Comment:
It seems this is *almost* the same as the shell form, except (1) different terminal name and (2) stripping quotation marks. That said, the code isn't very long so maybe the current approach is easiest to understand. Consider whether the existing organization is right, vs. finer-grained conditionals to deal with the two differences. The latter could be accomplished with subclasses: attribute with terminal name, function to strip quotes that's a no-op for shell form. Add a brief comment justifying your decision.